website/content/docs/configuration/index.mdx

```diff
@@ -76,6 +76,10 @@ to specify where the configuration is.
   Vault cluster. If omitted, Vault will generate a value. When connecting to
   Vault Enterprise, this value will be used in the interface.
 
+~> **Note**: It's recommended to set a cluster name explicitly rather than relying on it to be autogenerated
+since it helps better distinguish clusters & avoids issues related to
+Prometheus labels being misreported as blank values for `cluster`.
+
```

Comment:
I'm not sure this is a good resolution.

It pushes the work of setting this up on to every individual Vault admin, and worse, cluster name is completely irrelevant unless you're using Enterprise replication, so it's yet another thing for admins to care about, which provides very limited value to many.

I rather suspect that changing the configured cluster name for an existing cluster isn't well supported either, although that's not documented AFAIK.

If there is to be a specific reference to the Prometheus-specific labelling bug, it would be good to explicitly link to it - otherwise, there may well be a future state when the bug has been fixed, and people have to do historical research to understand the original reason for this recommendation.

Reply:
@maxb please see [here](https://hashicorp.atlassian.net/browse/VAULT-15467). This has also been raised as a ticket from a customer so this PR is a direct effect of that.

Reply:
Your JIRA isn't publicly visible.

Reply:
@maxb It's not meant to be public though, I just realized it. Anyway, this is a direct result of that and @jasonodonnell's [comment](https://github.com/hashicorp/vault/issues/11988#issuecomment-1545814204).